Wikipedia/Code/WMFLocationManager.h

```diff
@@ -19,6 +19,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (instancetype)coarseLocationManager;
```

Comment:
Make the initializer with the location manager public
To enable dependency injection